deepmd/pt/utils/stat.py

```diff
@@ -129,13 +136,67 @@ def compute_output_stats(
             )
         else:
             assigned_atom_ener = None
-        bias_atom_e, _ = compute_stats_from_redu(
-            merged_energy,
-            merged_natoms,
-            assigned_bias=assigned_atom_ener,
-            rcond=rcond,
-        )
+        if model is None:
+            # only use statistics result
+            bias_atom_e, _ = compute_stats_from_redu(
+                merged_energy,
+                merged_natoms,
```

Comment:
I noticed that this function is hardcoded with the `energy` key. Since we are refactoring, do we want to make it a bit more flexible, for example for `bias_dos`. Although, technically, we can hack the `bias_dos` to use `energy` as the keyword.

Reply:
We can use a new PR to add support for finetuning in other models.

Reply:
We can use a new PR to add support for finetuning in other models.